llvm/test/Transforms/InstCombine/cast-call-naked.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
```

Comment:
Use update_test_checks.py.

Preferably also find where all other tests for this transform are and add it to that file, instead of creating a new one.